examples/models/file/calendars.py

```diff
@@ -1,13 +1,16 @@
-from calendar import Calendar, day_abbr as day_abbrs, month_name as month_names
+from calendar import Calendar
+from calendar import day_abbr as day_abbrs
+from calendar import month_name as month_names
```

Comment:
This could be different with `--combine-as` but I think I prefer this, or better to update the example to not need "as" imports here